test_conformance/api/negative_platform.cpp

```diff
@@ -42,15 +42,17 @@ int test_negative_get_platform_info(cl_device_id deviceID, cl_context context,
 {
     cl_platform_id platform = getPlatformFromDevice(deviceID);
 
-    cl_int err =
-        clGetPlatformInfo(reinterpret_cast<cl_platform_id>(deviceID),
-                          CL_PLATFORM_VERSION, sizeof(char*), nullptr, nullptr);
-    test_failure_error_ret(
-        err, CL_INVALID_PLATFORM,
-        "clGetPlatformInfo should return CL_INVALID_PLATFORM  when: \"platform "
-        "is not a valid platform\" using a valid object which is NOT a "
-        "platform",
-        TEST_FAIL);
+    cl_int err(CL_SUCCESS);
+    for (auto invalid_platform : get_invalid_objects<cl_platform_id>(deviceID))
+    {
+        err = clGetPlatformInfo(invalid_platform, CL_PLATFORM_VERSION,
```

Comment:
Behavior when passing a null platform is implementation-defined, so we need to filter out nullptr objects here

Reply:
This is handled as a special case in `get_invalid_objects`, see https://github.com/KhronosGroup/OpenCL-CTS/pull/1148/files#diff-877e5e63eeec3f4e2314e73fb4c44f8e997bdfdc9e2fbb738798f2ef91082eebR201.

Reply:
yes indeed, thanks